### ISSUE TITLE
release 3.3.2?

### ISSUE THREAD

**Issue by bshaffer** [phpDocumentor/phpDocumentor#3434]
Can we tag/release version `3.3.2` (or whatever the next version should be)? I would like to use the fix from https://github.com/phpDocumentor/phpDocumentor/pull/3244 in my application!

Thank you!

**Comment by jaapio** [comment]
I would love to do that soon. But I'm pushing really hard to get some extra fixes in. Some of them are consuming way more time than I expected before. There is no planning but I do agree that it is about time to do a next release. 

I will have a look what the current status is and if we should skip some upcoming features to just have a new release ready.

**Comment by jaapio** [comment]
Did you know btw, that we are building a docker image every merge? 
https://hub.docker.com/layers/phpdoc/phpdoc/202301112131087e6028/images/sha256-3c6287e6207ad16bb82626f7b24d12416c3a8ec8de806cf2c355dfac0e1cd3a1?context=explore 

Contains you latest XML template change

**Comment by bshaffer** [comment]
Thanks! That's great! My use case requires building `phpdoc` as part of our own Docker image. It seems like that image does not use a phar, but instead installs using the whole repo. 

I tried copying this into our own repo like this:

```
COPY --from=phpdoc/phpdoc:latest /opt/phpdoc /opt/phpdoc
RUN ln -s /opt/phpdoc/bin/phpdoc /usr/local/bin/phpdoc
```

But I am getting a symphony error regarding a missing `.env` file, which I cannot find in the `phpdoc/phpdoc` image. If there was a phar I could copy, then I would be okay. Any thoughts on this?

Thanks again for your work on this repo!

**Comment by jaapio** [comment]
You need to set the run env for the application.  `ENV PHPDOC_ENV=prod`

**Comment by fibd** [comment]
@jaapio Is there any ETA on releasing 3.3.2 phar ? Its been quite a while now.

**Comment by mcserep** [comment]
Is there a planned release soon?
I noticed that some features documented at https://docs.phpdoc.org/3.0/ do not work, because simply they were not released yet. (The last release was made 14 months ago.)

Thanks!

**Comment by aprokopenko** [comment]
Is there a planned release soon?
Is there PHP 8.2 support will be released?

**Comment by jaapio** [comment]
Yes, I planned to do this when I'm back from vacation. Will be this month

**Comment by jaapio** [comment]
I just released v3.4.0. Thanks for your patience.
